src/main/java/com/azure/cosmos/examples/patch/sync/SamplePatchQuickstart.java

```diff
@@ -0,0 +1,599 @@
+                throughputProperties);
+        container = database.getContainer(containerResponse.getProperties().getId());
+        // </CreateContainerIfNotExists>
+
+        logger.info("Checking container " + container.getId() + " completed!\n");
+    }
+
+    private void createFamilies(List<Family> families) throws Exception {
+        double totalRequestCharge = 0;
+        for (Family family : families) {
+
+            // <CreateItem>
+            // Create item using container that we created using sync client
+
+            // Use lastName as partitionKey for cosmos item
+            // Using appropriate partition key improves the performance of database
+            // operations
+            CosmosItemRequestOptions cosmosItemRequestOptions = new CosmosItemRequestOptions();
+            CosmosItemResponse<Family> item = container.createItem(family, new PartitionKey(family.getLastName()),
+                    cosmosItemRequestOptions);
+            // </CreateItem>
+
+            // Get request charge and other properties like latency, and diagnostics
+            // strings, etc.
+            logger.info(String.format("Created item with request charge of %.2f within duration %s",
+                    item.getRequestCharge(), item.getDuration()));
+
+            totalRequestCharge += item.getRequestCharge();
+        }
+        logger.info(String.format("Created %d items with total request charge of %.2f", families.size(),
+                totalRequestCharge));
+
+        Family family_to_upsert = families.get(0);
+        logger.info(String.format("Upserting the item with id %s after modifying the isRegistered field...",
+                family_to_upsert.getId()));
+        family_to_upsert.setRegistered(!family_to_upsert.isRegistered());
+
+        CosmosItemResponse<Family> item = container.upsertItem(family_to_upsert);
+
+        // Get upsert request charge and other properties like latency, and diagnostics
+        // strings, etc.
+        logger.info(String.format("Upserted item with request charge of %.2f within duration %s",
+                item.getRequestCharge(), item.getDuration()));
+    }
+
+    // demonstrates a single patch (add) operation
+    private void patchAddSingle(String id, String partitionKey) {
+        logger.info("Executing Patch with single 'add' operation");
+
+        CosmosPatchOperations cosmosPatchOperations = CosmosPatchOperations.create();
+        // attribute does not exist, will be added
+        cosmosPatchOperations.add("/vaccinated", false);
+
+        CosmosPatchItemRequestOptions options = new CosmosPatchItemRequestOptions();
+        try {
+            CosmosItemResponse<Family> response = this.container.patchItem(id, new PartitionKey(partitionKey),
+                    cosmosPatchOperations, options, Family.class);
+            logger.info("Item with ID " + response.getItem().getId() + " has been patched");
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
we should use the logger everywhere instead of e.printStackTrace().
fix these everywhere please.

```suggestion
            logger.error("failed", e);
```

Reply:
done